lib/builders/latexmk.js

```diff
@@ -49,21 +51,22 @@ export default class LatexmkBuilder extends Builder {
   }
 
   constructArgs (filePath, jobname) {
-    const outputFormat = atom.config.get('latex.outputFormat') || 'pdf'
-
     const args = [
       '-interaction=nonstopmode',
       '-f',
       '-cd',
-      `-${outputFormat}`,
       '-file-line-error'
     ]
 
+    const outputFormat = this.getOutputFormatFromMagic(filePath) ||
+      atom.config.get('latex.outputFormat') || 'pdf'
```

Comment:
Extract this fall-through logic into a method. E.g. 
```js
getOutputFormat (filePath) {
  const outputFormat = this.getOutputFormatFromMagic(filePath)
  if (outputFormat) return outputFormat

  return atom.config.get('latex.outputFormat')
}
```

I don't understand why the `|| 'pdf'` part was added? We should use whatever default output format the user has configured? The default value is `pdf` by the way.